plugin/admin/options/option-types/class-selectfield.php

```diff
@@ -37,28 +37,43 @@ public function register() {
 	 * @return int Sanitized value.
 	 */
 	public function sanitize( $value ) {
-		if ( 'pixels' === $value ) {
-			return 'pixels';
-		}
-
-		if ( 'percentage' === $value ) {
-			return 'percentage';
-		}
```

Comment:
This is all incorrect, am I getting it right? It's verifying against values which are not even in the options (there should be `list` and `grid`)

Reply:
Yes, you right - this is wrong code.

Reply:
Ok, well, thanks for noticing & fixing it.